internal/rest/avi_evh_obj.go

```diff
@@ -430,6 +430,9 @@ func (rest *RestOperations) AviVsBuildForEvh(vs_meta *nodes.AviEvhVsNode, rest_m
 		}
 		vs.AnalyticsPolicy = vs_meta.GetAnalyticsPolicy()
 
+		if vs_meta.NetworkSecurityPolicyRef != nil {
+			vs.NetworkSecurityPolicyRef = vs_meta.NetworkSecurityPolicyRef
```

Comment:
I think we are adding this field twice since it should be copied with the generated fields in line 436.